Mathlib/MeasureTheory/SpecificCodomains/ContinuousMap.lean

```diff
@@ -0,0 +1,129 @@
+/-
+Copyright (c) 2025 Anatole Dedecker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anatole Dedecker
+-/
+import Mathlib.Topology.ContinuousMap.Compact
+import Mathlib.Topology.ContinuousMap.Algebra
+import Mathlib.MeasureTheory.Integral.IntegrableOn
+
+/-!
+# Specific results about `ContinuousMap`-valued integration
+
+In this file, we collect a few results regarding integrability, on a measure space `(X, μ)`,
+of a `C(Y, E)`-valued function, where `Y` is a compact topological space and `E` is a normed group.
+
+These are all elementary from a mathematical point of view, but they require a bit of care in order
+to be conveniently usable. In particular, to accomodate the need of families `f : X → Y → E` which
+such that `f x` is only continuous for *almost every* `x`, we give a variety of results about the
+integrability of `fun x ↦ ContinuousMap.mkD (f x) g` whose assumption only mention `f` (so that
+user don't have to convert between `f` and `fun x ↦ ContinuousMap.mkD (f x) g` by hand).
+
+## Main results
+
+* `hasFiniteIntegral_of_bound`: given `f : X → C(Y, E)`, the natural way to show
+`HasFiniteIntegral f` is to give a `bound : X → ℝ`, which itself has finite integral, and such that
+`∀ᵐ x ∂μ, ∀ y : Y, ‖f x y‖ ≤ bound x`.
```

Comment:
Do the continued lines parse properly in markdown without indentation? I suspect they don't, but I could be wrong. In any case, I would probably indent everywhere (not just on this bullet) for the sake of the source.